app/models/grouping.rb

```diff
@@ -83,7 +84,7 @@ class Grouping < ApplicationRecord
 
   has_many :grouping_starter_file_entries, dependent: :destroy
   has_many :starter_file_entries, through: :grouping_starter_file_entries
-
+  validate :test_student_grouping_member, if: -> { !self.id.nil? && self.test_students.exists? }
```

Comment:
use `new_record?` instead of checking for nil.

also, can we do this validation with having a `test_students` association please.

also, this validation should be doable even if we have a new record